src/System.ServiceModel.Security/ref/System.ServiceModel.Security.cs

```diff
@@ -22,6 +22,7 @@ public abstract partial class MessageSecurityVersion
         public static System.ServiceModel.MessageSecurityVersion WSSecurity10WSTrustFebruary2005WSSecureConversationFebruary2005WSSecurityPolicy11BasicSecurityProfile10 { get { return default; } }
         public static System.ServiceModel.MessageSecurityVersion WSSecurity11WSTrustFebruary2005WSSecureConversationFebruary2005WSSecurityPolicy11 { get { return default; } }
         public static System.ServiceModel.MessageSecurityVersion WSSecurity11WSTrustFebruary2005WSSecureConversationFebruary2005WSSecurityPolicy11BasicSecurityProfile10 { get { return default; } }
+        public static System.ServiceModel.MessageSecurityVersion WSSecurity11WSTrust13WSSecureConversation13WSSecurityPolicy12BasicSecurityProfile10 { get { return default; } }
```

Comment:
what is the 'default' here?

Reply:
This is just the reference assembly. Using `default` produces the smallest IL for the reference assembly. The implementation was already there in S.Private.SM